library/Icinga/Web/Menu.php

```diff
@@ -130,5 +130,16 @@ public function init()
                 'priority'    => 900
             ]));
         }
+
+        if (Auth::getInstance()->hasPermission('manage/rememberme')) {
+            $this->getItem('system')->addChild($this->createItem('Logged_in_users', [
+                'icon'        => 'host',
+                'description' => t('List of users who stay logged in'),
+                'label'       => t('Logged in users'),
+                'url'         => 'manageuserdevices',
+                'permission'  => 'manage/rememberme',
+                'priority'    => 890
+            ]));
+        }
```

Comment:
* Please add the menu entry just like the other entries are added (directly in the array definition)
* Drop the explicit permission check, you're already passing the permission as config option which suffices
* Use the label `User Sessions` instead
* Remember to use the adjust permission name